README.md

```diff
@@ -11,175 +11,117 @@ An opinionated JWT library with sensible defaults that supports refresh and acce
 npm install authomatic
 ```
 
+## Available stores
+[Redis](https://github.com/wearereasonablepeople/authomatic-redis)
+
+Please create an issue if you need another store.
+
+## Examples
+[Koa Example](/examples/koa.js)
+
+## Quickstart
+```javascript
+const secret = 'thisIsAVeryBadSecret';
```

Comment:
The secret does not appear to be used. I realise that it's because it's used as an argument to the methods. Maybe we should remove it from the example, or change the API so that the secret is passed as an option. (Why didn't we do that?)

Reply:
I should remove it from the readme then.

It is not passed as an option because it is very possible for secrets to change in run time.

Reply:
If they do change at runtime, I would just create a new Authomatic instance.

Reply:
as discussed offline